src/poliastro/twobody/orbit.py

```diff
@@ -216,12 +216,29 @@ def h_vec(self):
         h_vec = np.cross(self.r.to(u.km).value, self.v.to(u.km / u.s)) * u.km ** 2 / u.s
         return h_vec
 
+    @property
+    def h_mom(self):
+        """Angular momentum. """
+        h_mom = np.linalg.norm(self.h_vec) * u.km ** 2
```

Comment:
Instead of using `np.linalg.norm`, you can use `poliastro.util.norm`, which respects the units